### ISSUE TITLE
cmd/asm: update where NOFRAME is valid

### ISSUE THREAD

**Issue by randall77** [golang/go#38110]
The current comment for `NOFRAME` in `runtime/textflag.h` says:
```
// TODO(mwhudson): only implemented for ppc64x at present.
```
It's been implemented for lots (all?) architectures since.
Check our coverage and update (delete) this comment.

@mwhudson @zhangfannie

**Comment by cherrymui** [comment]
Yeah, I think it is implemented in nearly all architectures. If I remember correctly, @aclements did that.

**Comment by cherrymui** [comment]
CL https://go-review.googlesource.com/c/go/+/92039 and the stack.

**Comment by mwhudson** [comment]
Oh yay, glad that got done!
